src/components/Accordion/AccordionData.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable max-len */
+export const accordionItemsData = [
+  {
+    id: 1,
+    title: 'description',
+    description:
+      ' Architecto, earum temporibus quidem ex eaque unde harum sit, deleniti tempore laboriosam neque cumque!Architecto, earum temporibus quidem ex eaque unde harum sit, deleniti tempore laboriosam neque cumque!',
+  },
+  {
+    id: 2,
+    title: 'additional info',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime vel sit possimus laborum error eius. Maxime vel sit possimus laborum error eius. Earum temporibus quidem ex eaque unde harum sit, deleniti tempore laboriosam neque cumque! ',
+  },
+  {
+    id: 3,
+    title: 'tags',
+    description: 'Earum temporibus quidem ex eaque unde harum sit, deleniti tempore laboriosam neque cumque!Earum temporibus quidem ex eaque unde harum sit, deleniti tempore laboriosam neque cumque!',
+  },
+];
```

Comment:
Do not store any data inside of component folder, use some other place for this. Like **mocks** or **config** directory